src/EnergyPlus/MixedAir.cc

```diff
@@ -3279,6 +3276,155 @@ namespace MixedAir {
 						SetupEMSActuator( "Outdoor Air Controller", OAController( OAControllerLoop ).Name, "Air Mass Flow Rate", "[kg/s]", OAController( OAControllerLoop ).EMSOverrideOARate, OAController( OAControllerLoop ).EMSOARateValue );
 					}
 
+					VentMechObjectNum = OAController(OAControllerLoop).VentMechObjectNum;
```

Comment:
@JasonGlazer If you haven't found this yet, it's failing in line 3446 because VentMechObjectNum is not initialized there.  This line of code needs to be repeated down there (using OAControllerNum), or added back at the top where it was.  We need it in both places, because this block is looping over all OA controllers, but other parts of this routine are working on only the current OA controller.  I can push up the fix if you want.